modules/ROOT/pages/_partials/metrics/monitoring-sync.adoc

```diff
@@ -1,49 +1 @@
-
-
-= Monitoring {sync-service} Service
-
-== Prerequisites
-
-* [x] You have provisioned the xref:mobile-metrics.adoc[{metrics-service} service].
-* [x] You have provisioned the xref:data-sync.adoc[{sync-service}] service.
-
-NOTE: The {metrics-service} Service and {sync-service} Service must be provisioned in the same OpenShift project to access data.
-
-== Overview
-
-After the {metrics-service} Service (includes Grafana for visualization and the Prometheus monitoring system) and {sync-service} Service are provisioned,
-you should be able to see the "Data Sync Metrics" in the list of available dashboards (navigate to Grafana's exposed URL -> Log in -> Home -> Select *Data Sync Metrics*).
-
-== Dashboard panel descriptions
-
-The Data Sync dashboard consists of several panels which give you an overview of the specific
-events and resources, such as memory usage, CPU usage, resolved operation count etc.
-
-Below you will find a detailed description of each panel and its values.
-
-== Singlestat Panels
-
-Singlestat panels show you the main summary of a single data series.
-
-* *CPU Usage*: CPU usage percentage of {sync-service} Service used currently.
-
-* *Memory Usage*: The amount of memory currently used by the {sync-service} Service
-
-== Graph panels
-
-Used to show how certain values change over time, e.g. the number of operations resolved.
-
-* *Memory Usage in Time*: The amount of memory used by the {sync-service} Service over time
-
-* *CPU Usage in Time*: The percentage of CPU used by the {sync-service} Service over time
-
-* *Queries/Mutations Resolved*: Overview of the resolved queries and mutations over time
-
-* *Resolver Timings*: Overview of the how long it took to resolve operations over time.
-
-
-== Additional resources
-
-* link:http://docs.grafana.org/features/panels/singlestat/#singlestat-panel[Singlestat Panel]
-* link:http://docs.grafana.org/features/panels/graph/#graph-panel[Graph Panel]
-* link:https://grafana.com/plugins/grafana-piechart-panel[Pie Chart]
+include::../sync/server-metrics-partial.adoc[]
```

Comment:
this will have knock on effects for downstream docs, any reason why we're moving it out of metrics?
If we do it for sync, should we also do it for IDM, right? (or maybe I'm not understanding motivation here)

Reply:
@finp 
We have 2 places where we talk about sync metrics.
1. Metrics service docs talk about how to check sync metrics
2. Sync service docs talk about how to check sync metrics

They were duplicated previously and now I created a partial and included in those 2 places.

> If we do it for sync, should we also do it for IDM, right? (or maybe I'm not understanding motivation here)

You're right about IDM. There are also 2 metrics docs for IDM:
* https://deploy-preview-631--aerogear-docs-preview.netlify.com/aerogear/latest/mobile-metrics/#identity
* https://deploy-preview-631--aerogear-docs-preview.netlify.com/aerogear/latest/identity-management/#monitoring
and the source is duplicated:
* https://github.com/wojta/mobile-docs/blob/8e614ac3cbd3552a7e4561057e804102d606d3ee/modules/ROOT/pages/_partials/keycloak/monitoring-idm.adoc
* https://github.com/aliok/mobile-docs/blob/6d380e0c617d3c818a291dbd76129487c26d84d7/modules/ROOT/pages/_partials/metrics/monitoring-keycloak.adoc

@finp I will create a new partial for this too, if that's ok?

> this will have knock on effects for downstream docs, any reason why we're moving it out of metrics?
Don't know what those effects are

Reply:
right, don't worry about downstream, but I'm not sure i understand the partials you propose?

Reply:
let me do it and it will be more clear

Reply:
@finp I checked Keycloak things I mentioned above. It turns out one of the duplicates was not used. Now I deleted it.

Reply:
@aliok  I think this might cause issues, given the changes to IDM service docs.. You're merging into sync branch, does it make sense to limit edits to 'sync stuff'?

Reply:
Ok, reverted all changes except the sync metrics one